docs/Update-MarkdownHelpSchema.md

```diff
@@ -15,20 +15,18 @@ Update-MarkdownHelpSchema [-Path] <String[]> [-OutputFolder] <String> [[-Encodin
 ```
 
 ## DESCRIPTION
-Migrate to the latest markdown help schema.
+The **Update-MarkdownHelpSchema** cmdlet migrates markdown help files to the latest PlatyPS markdown help schema.
+We recommend you update to the latest schema.
 
-As a release 0.4.0, there are two schemas: 1.0.0 and 2.0.0
-We highly encourage you to migrate to the latest schema.
-
-It's easlier and provides visually apealing expirience.
+As of release 0.4.0, there are two schemas: 1.0.0 and 2.0.0.
```

Comment:
We probably should remove release mention, it would be a headache to keep it up to date.